README.md

```diff
@@ -1 +1,54 @@
-# Hello-Microverse
\ No newline at end of file
+# 📖 Hello Microverse 
+
+Hello Microverse is a test project built to understand the basics of professional git workflows.
+
+## 🛠 Built With
```

Comment:
- [ ] - [ ] Kindly please use [this Microverse template](https://github.com/microverseinc/readme-template) for your Readme file. This will give more details about your project. 💁‍♂️